release-team/role-handbooks/branch-manager/README.md

```diff
@@ -1,235 +1,307 @@
+Adding the `--rc` flag switches behavior on to building release candidates. Again `gcbmgr` and `anago` automatically finds and increments the current build number. For example:
 
-2. `rc.1` should be staged and released
+`./gcbmgr stage release-1.12 --rc --build-at-head --nomock`
 
-   Make sure that all the changes that have been merged in master also made it
-   into the release branch. You can also do a
-   [`branchff`](#branch-fast-forward) to see the state of the two branches and
-   potentially pull any remaining PR from master into the release branch.
+To publish the build artifacts (release), as usual use the `--buildversion=` number as specified in the output when `gcbmgr` is done with the stage command.
 
-   At this point in time, the master and the release branch are the same.
-   Nothing new gets merged into master, because we are still in code freeze.
-   Therefore, it is safe to cut the release candidate.
+In an perfect world, `rc.1` and the official release are the same commit. To get as close to that perfect state as we can, the following things should be considered:
 
-3. Code Freeze can be lifted
+1. PRs tagged with the release cycle milestone should have all merged onto the `master` branch:
 
-   Technically we can keep code freeze in place after `rc.1` was cut. However,
-   we should aim at lifting code freeze relatively quickly after `rc.1`.
+   For example, there should be no open PRs tagged with the v1.14 milestone, [`is:pr is:open milestone:v1.14`][pr-milestone-query] .
 
-   Otherwise we might have a mix of PRs against master, some have been merged
-   in code freeze and for the milestone, just after `rc.1`, and others have
-   been merged when code freeze has been lifted. We might miss this specific PR
-   in the plethora of PRs that [tide] merges after code thaw, and we might
-   miss that this PR actually needs to be cherry-picked into the release
-   branch.
+   This means you and the release team should push for PRs to get merged before the code freeze is lifted or determine if the open PRs can be removed from the milestone and be merged in the next release cycle.
+
+2. When to staged and release `rc.1`:
+
+   Make sure that all the changes that have been merged onto `master`, make it onto the release branch. Run [`./branchff`](#branch-fast-forward to see the state of the two branches and pull any remaining PRs from `master` onto the release branch.
+
+   At this point in time, `master` and the release branch should have the same commits. Nothing new gets merged onto `master`, since code freeze is not lifted. Therefore, it is safe to cut the release candidate.
+
+3. Factors that determine if Code Freeze can be lifted:
+
+   - Zero pending PRs and no open issues tagged with the release cycle milestone.
+   - No failing X.Y-blocking tests, for example the [sig-release-1.15-blocking](https://testgrid.k8s.io/sig-release-1.15-blocking) dashboard should not have any red failing test builds.
+
```

Comment:
One more for the list is something like "If pending PRs, open issues, or failing test counts are non-zero, there is high confidence that the issue is understood and can be remedied via a cherry pick to the release branch between the release candidate(s) and official release."